src/tcgwars/logic/impl/gen4/SupremeVictors.groovy

```diff
@@ -2871,464 +3507,648 @@ public enum SupremeVictors implements LogicCardInfo {
           onPlay {
+            flip {
+              my.hand.getExcludedList(thisCard).moveTo(hidden:true,my.deck)
+              my.deck.select("Search your deck for a card").moveTo(hidden:true,my.hand)
+              shuffleDeck()
+            }
           }
           playRequirement{
+            assert my.hand.getExcludedList(thisCard) || my.deck : "You have no other cards in your hand or deck"
           }
         };
       case PALMER_S_CONTRIBUTION_139:
-        return basicTrainer (this) {
-          text "You can play only one Supporter card each turn. When you play this card, put it next to your Active Pokémon. When your turn ends, discard this card.\nSearch your discard pile for up to 5 in any combination of Pokémon and basic Energy cards. Show them to your opponent and shuffle them into your deck."
+        return supporter (this) {
+          text "You can play only one Supporter card each turn. When you play this card, put it next to your Active Pokémon. When your turn ends, discard this card." +
+            "Search your discard pile for up to 5 in any combination of Pokémon and basic Energy cards. Show them to your opponent and shuffle them into your deck."
           onPlay {
+            my.discard.select(max:5,"Search your discard pile for up to 5 Pokémon and basic Energy cards",{it.cardTypes.is(POKEMON)||it.cardTypes.is(BASIC_ENERGY)})
           }
           playRequirement{
+            assert my.discard.filterByType(POKEMON) || my.discard.filterByType(BASIC_ENERGY) : "You have no Pokémon or basic Energy cards in your discard pile"
           }
         };
       case VS_SEEKER_140:
-        return basicTrainer (this) {
-          text "Search your discard pile for a Supporter card, show it to your opponent, and put it into your hand."
-          onPlay {
-          }
-          playRequirement{
-          }
-        };
+        return copy(FireRedLeafGreen.VS_SEEKER_100, this);
       case ABSOL_G_LV_X_141:
-        return levelUp (this, from:"Absol G", hp:HP100, type:DARKNESS, retreatCost:1) {
+        return levelUp (this, from:"Absol G", hp:HP100, type:D, retreatCost:1) {
           weakness F
           resistance P, MINUS20
           pokePower "Darkness Send", {
-            text "Once during your turn , when you put Absol LV. from your hand onto your Active Absol , you may flip 3 coins. For each heads, put the top card from your opponent’s deck in the Lost Zone."
-            actionA {
+            text "Once during your turn , when you put Absol LV.X from your hand onto your Active Absol , you may flip 3 coins. For each heads, put the top card from your opponent’s deck in the Lost Zone."
+            onActivate {r->
+              if (r==PLAY_FROM_HAND && opp.deck && confirm('Use Darkness Send?')) {
+                powerUsed()
+                def count
+                flip 3, {
+                  count ++
+                }
+                opp.deck.subList(0,count).moveTo(opp.lostZone)
```

Comment:
I don't think it's necessary to keep a count, you can just move each card as the heads happens since this is an ability.

Reply:
I did this to avoid chat log spam. Is there any reason to change this beyond aesthetic?

Reply:
That makes sense. No real reason, no.